test/system/252-quadlet.bats

```diff
@@ -1013,4 +1013,81 @@ EOF
     run_podman rmi $(pause_image)
 }
 
+@test "quadlet - image files" {
+    local quadlet_image_unit=image_test_$(random_string).image
+    local quadlet_image_file=$PODMAN_TMPDIR/$quadlet_image_unit
+    cat > $quadlet_image_file <<EOF
+[Image]
+Image=$IMAGE
+EOF
+
+    # Remove the local image to make sure it will be pulled again
+    run_podman image rm $IMAGE
+
```

Comment:
We should avoid repulling as much as possible. Podman pull supports the transports as documented in `containers-transports(5)`, so you could just "pull" from a local directory to avoid network flakes.

Reply:
What @Luap99 said. If it's too difficult to pull from dir or local registry, please use `$PODMAN_NONLOCAL_IMAGE_FQN` and do not delete `$IMAGE`.

Reply:
The setup for pulling from a local directory seemed a bit cumbersome to me. So, for now, I moved to use the `NONLOCAL` image. I hope that's fine